codec-http/src/main/java/io/netty/handler/codec/spdy/SpdyCodecUtil.java

```diff
@@ -305,9 +305,6 @@ static void validateHeaderName(CharSequence name) {
                 throw new IllegalArgumentException(
                         "name contains null character: " + name);
             }
-            if (c >= 'A' && c <= 'Z') {
```

Comment:
@buchgr - It was pulled out into https://github.com/netty/netty/pull/4047 so we can easily get it to all branches.